packages/contentful-migrations/ui/1100000000004-create-m-columns-migration.js

```diff
@@ -37,8 +33,8 @@ module.exports = async function (migration, context) {
     .defaultValue({
       [defaultLocale.code]: 'light',
```

Comment:
we could remove the defaultValue as it is disabled & omitted.

Reply:
done